lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -46,6 +46,7 @@ export class RequestUtils {
         if(request) {
             // if extraScopesToConsent is passed in loginCall, append them to the login request; Validate and filter scopes (the validate function will throw if validation fails)
             scopes = isLoginCall ? ScopeSet.appendScopes(request.scopes, request.extraScopesToConsent) : request.scopes;
+            scopes = scopes ? scopes.map(scope => scope.trim()) : scopes;
```

Comment:
Could this use the helper function you defined above?

Reply:
The helper is in ServerRequestParameters, not in RequestUtils

Reply:
Right, could it go in a place where both can use it?

Reply:
e.g. in the `ScopeSet` class

Reply:
Made some updates, take a look.